src/utils/ohdf/types.ts

```diff
@@ -45,9 +45,9 @@ export type DataOrArray = Data | Data[] | PrintableSummary | PrintableSummary[];
 
 /**
  * Represents the possible values for the row names in the generated Markdown table.
- * The possible values are 'Total', 'Critical', 'High', 'Medium', 'Low', and 'Not Applicable'.
+ * The possible values are 'Total', 'Critical', 'High', 'Medium', 'Low', and 'None'.
  */
-export type RowType = 'total' | 'critical' | 'high' | 'medium' | 'low' | 'Not Applicable';
+export type RowType = 'total' | 'critical' | 'high' | 'medium' | 'low' | 'none';
```

Comment:
i'm not familiar enough with threshold files to know if the changes you're making here are appropriate.  please get @aaronlippold / @wdower / @ejaronne to confirm if this is alright.

Reply:
Had a discussion with @aaronlippold and @wdower regarding this table. There is no need to have a row for severity "none" because it should not be possible for there to be severity "none". Every mapper should define a severity, especially in the case when status is not applicable, because then impact would be 0 leading to severity "none".

I have reverted my changes, but the option for `RowType: 'Not Applicable'` has been removed because it is never used and does not need to be used because there is a column that represents this. 

The script that generates [this](https://github.com/mitre/redhat-enterprise-linux-9-stig-baseline/actions/runs/9863765970) kind of table may need to be updated to reflect the possibility of there being N/A controls with severity critical, high, medium, or low.

Reply:
Please make a new issue for this task @kemley76